src/garage/np/algos/cem.py

```diff
@@ -62,6 +70,11 @@ def __init__(self,
         self.n_params = None
         self._initialize()
 
+        if policy.vectorized:
```

Comment:
Do we even have any non-vectorized policies? I think we should just always use `BatchSampler`.

Reply:
All policies are now vectorized, AFAICT, so this check can be removed